apps/client/src/features/table/utils.js

```diff
@@ -39,7 +39,7 @@ export const makeTable = (headerData, tableData, userFields) => {
   const data = [
     ['Ontime · Schedule Template'],
     ['Event Name', headerData?.title || ''],
-    ['Event URL', headerData?.url || ''],
+    ['Public URL', headerData?.publicUrl || ''],
```

Comment:
Are we not missing a backstage url here?

Reply:
i have thought about adding it there but decided against it. Maybe the export is shared with third parties that may not have access to the backstage url.

Reply:
We don't really protect the data in the app. The assumption is that anyone that has access to the page should be entitled to the data

Reply:
alright